### ISSUE TITLE
Volume Characteristic Warnings in v3.7.17

### ISSUE THREAD

**Issue by tjm7844** [grzegorz914/homebridge-denon-tv#75]
Since updating to v3.7.17, these warnings are being generated on the update interval (stack trace from debug mode included):

_[2/25/2021, 11:35:19 AM] [homebridge-denon-tv] This plugin generated a warning from the characteristic 'Volume': characteristic was expected valid number and received NaN. See https://git.io/JtMGR for more info.
[2/25/2021, 11:35:19 AM] [homebridge-denon-tv] Error: 
    at Volume.Characteristic.characteristicWarning (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2164:105)
    at Volume.Characteristic.validateUserInput (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1992:16)
    at Volume.Characteristic.updateValue (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1365:20)
    at TelevisionSpeaker.Service.updateCharacteristic (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Service.ts:465:35)
    at denonTvDevice.updateDeviceState (/usr/lib/node_modules/homebridge-denon-tv/index.js:262:24)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)_

**Comment by grzegorz914** [comment]
Try 3.7.19

**Comment by tjm7844** [comment]
Sorry for the delay - Homebridge was telling me there's an update available, but it wouldn't actually update. Had to force it to 3.7.19 through the terminal. Homebridge UI is now showing 3.7.19, but I'm still getting the same error.

**Comment by grzegorz914** [comment]
Can U be more specific in which situations this happened, on my side no any warning. May be this happened if volume is set to 0?

**Comment by grzegorz914** [comment]
Please also get the output from:
http://addresipreceiver:8080/goform/formMainZone_MainZoneXmlStatusLite.xml

**Comment by tjm7844** [comment]
I think I may have corrected the issue - the error is not displaying any more, at least. For context, the way I was configured I had a Main Zone with volume control, and then a Zone 2 that feeds another amplifier. To that end, I had Volume Control set to "None" for Z2 (but the volume always stays at 0 on my Denon).

I went into the config, turned on the Volume slider in homebridge, restarted homebridge, and then fiddled around with the volume slider a bit. The homebridge logs reflected accurately, and stopped giving me the error. I've since gone and set the volume back to 0 and turned off the slider for Z2, and am still not getting the error message.

Don't know what initially caused it (it had never happened before), but looks like something I did in that process fixed it. I'm happy to close the issue if you'd like (and thank you for the quick responses!).
